vector/src/main/java/fr/gouv/tchap/activity/TchapRoomCreationActivity.java

```diff
@@ -419,4 +447,18 @@ public void onClick(DialogInterface dialog, int which) {
                 })
                 .show();
     }
+
+    /**
+     * Open the screen to select the members to invite in the room.
+     *
+     * @param requestCode    the request id.
+     */
+    private void inviteMembers(int requestCode) {
+        Intent intent = new Intent(TchapRoomCreationActivity.this, VectorRoomInviteMembersActivity.class);
+        intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_MATRIX_ID, mSession.getMyUserId());
+        intent.putExtra(VectorRoomInviteMembersActivity.EXTRA_HIDDEN_PARTICIPANT_ITEMS, mParticipants);
```

Comment:
We don't have to provide here a `mParticipants`
Please remove this input in the intent